### DISCUSSION TITLE
5D2GwB4TZ9rHMiuX3vQKcSn5gj93mREQVFXGxvTxPXB7jKmN

### DISCUSSION THREAD

**Discussion by magicstone1412** [entropyxyz/community#37] [Get Test Funds] [answered]
### Address

5D2GwB4TZ9rHMiuX3vQKcSn5gj93mREQVFXGxvTxPXB7jKmN

**Comment by magicstone1412** [comment]
I would like to receive 10,000 test funds to setup a validator for the coming period. Thank you.

**Comment by johnnymatthews** [comment, marked as answer]
Just sent you some funds. You won't be able to set up a validator node just yet, but you can still play around with the network.
